### PR TITLE
Week1 homework website what imiss about munich

### PR DESCRIPTION
**Pull request by stankasvecova**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+              <div class="container"></div>
+              <div class="card_imgWrapper">
+                   <img class="card_img" src="nordbad.jpg">
+              </div>
+              <div class="card_description"><br>Nordbad is a fun and relaxing place where to go on a weekend </br>when there's bad weather and you find it <a href="https://www.google.com/maps/place/Nordbad/@48.1603281,11.5615484,17z/data=!3m1!4b1!4m5!3m4!1s0x479e75deb97b8337:0x6bca64d0f3009f64!8m2!3d48.1603245!4d11.5637424">here</a>
+              </div>
+          </section>
+
+          <section class="card">
+            <h2 class="card_title">Skiing nearby Munich</h2>
```
Comment (BTBTravis) [inline index.html:31]: I also miss skiing 
Screenshot 2020-04-19 at 11 31 39

File: index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            <div class="container"></div>
+            <div class="card_imgWrapper">
+                <img class="card_img" src="skiingnearmunich.jpg">
+            </div>
+            <div class="card_description"><br>Brauneck or any other nearby skiing places are great </br>for fun day-trips full of fresh air and sports. Get a train or drive a car <a href="https://www.google.com/maps/place/Ski-+und+Wandergebiet+Brauneck/@47.6636382,11.4510302,12z/data=!4m13!1m7!3m6!1s0x479d97800959c6d9:0x1d1e48aecf891b10!2sBrauneck!3b1!8m2!3d47.6636111!4d11.5211111!3m4!1s0x479d90a53dbec0d5:0x541d092224075f!8m2!3d47.6748663!4d11.5545273">here</a>
+            </div>
+        </section>
+        
+        <section class="card">
+            <h2 class="card_title">Fit One Gym</h2>
```
Comment (BTBTravis) [inline index.html:39]: I see a lot of classes that are not being used in your css file. Class attributes on html elements should only be added if they have a purpose. I would remove the ones you don't have css rules for to make things cleaner.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    top: 0;
+    left: 0;
+    right: 0;
+    height:8px;
+}
+.main {
+  display: flex;
+  flex-direction: column;
+  padding: 2em 20vw;
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
```
Comment (BTBTravis) [inline styles.css:13]: You have used 'Lucida Sans' here but you are not providing this font in your html file. So most users won't have it but you have lots of fallbacks so this is not a problem just making sure you know.

**Review by BTBTravis — COMMENTED** [review]
nice work @stankasvecova  I left some cleanup comments nothing big.

**Review by BTBTravis — CHANGES_REQUESTED** [review]
Requesting changes of removing the unused css classes from html

File: index.html (hunk 1 above)
Comment (stankasvecova) [reply index.html:31]: :) I know from your website (I checked out last night and it's cool :) that you are an avid skier and climber! Any tips on where to go climb in the nature around Munich in these times? Thanks

File: index.html (hunk 2 above)
Comment (stankasvecova) [reply index.html:39]: True, thanks. Will edit.

File: styles.css (hunk 3 above)
Comment (stankasvecova) [reply styles.css:13]: I see! Thanks for pointing out.

**Pull request merged**